docs/src/pages/components/slider/VerticalAccessibleSlider.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/core/Slider';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  forceWebkitVerticalOrientation: {
+    '& input[type="range"]': {
+      '-webkitAppearance': 'slider-vertical',
+    },
+  },
+});
+
+export default function VerticalSlider() {
+  const classes = useStyles();
+
+  function preventHorizontalKeyboardNavigation(event: React.KeyboardEvent) {
```

Comment:
```suggestion
  const preventHorizontalKeyboardNavigation = (event: React.KeyboardEvent) => {
```

Reply:
What is the point of this comment?

Reply:
The objective would be to have the demos as consistent as possible.

Reply:
Sure, propose this code style via issue/PR and accomodate it with a rationale and lint rule. Anything short of that is review noise.

Reply:
Not everything has to be inside a lint rule

Reply:
So we agree that it doesn't matter. Let's move on then.